### PR TITLE
Add blog post on importance of open source software

### PR DESCRIPTION
**Pull request by Fashander**
# Description

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/blog/2024-09-13-why-open-source-important-proprietary-uncertainties.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+![Why open source software is important vs proprietary uncertainties](/img/blog/open-source-proprietary-risks.jpg)
+
+A primary downside of non-open source solutions (proprietary, SSPL, and other open source pretenders) is that you're always at risk, at all times.
+At any given moment, that software you depend on may be deprecated – and you're left with months or years of wasted effort.
+
+<!--truncate-->
+
+That was [the case with MongoDB over the past week with the deprecation of Data API and Device Sync](https://www.mongodb.com/docs/atlas/app-services/deprecation/#std-label-app-services-deprecation), among other features.
+
+A primary downside of non-open source solutions (proprietary, SSPL, and other open source pretenders) is that you're always at risk, at all times.
```
Comment (AlekSi) [inline website/blog/2024-09-13-why-open-source-important-proprietary-uncertainties.md:19]: We have the same sentence above

File: website/blog/2024-09-13-why-open-source-important-proprietary-uncertainties.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+The eventual fate of certain features are not decided by a single vendor.
+
+In contrast to proprietary solutions where you're always in danger of lock-in, open source software like FerretDB does not lock you into a specific ecosystem.
+
+FerretDB offers a truly open-source MongoDB alternative database built on top of PostgreSQL.
+You're free to host FerretDB on your own infrastructure, either on-premise, cloud, or hybrid – total control over your data and operations.
+
+## FerretDB replacement for Data API
+
+In light of the recent deprecation announcement**, [we have started working on an alternative for Data API](https://github.com/FerretDB/FerretDB/discussions/4578).** The first iteration of that HTTP API should be out in a few weeks, along with the first public version of FerretDB v2.
```
Comment (AlekSi) [inline website/blog/2024-09-13-why-open-source-important-proprietary-uncertainties.md:62]: Those `**` do not render correctly

File: website/blog/2024-09-13-why-open-source-important-proprietary-uncertainties.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+When a feature like the Data API is removed, alternative options are quite limited.
+
+## Why open source solutions are important
+
+Community is at the forefront of everything in open source!
+Every feature, change, and development is done to meet the needs of the user.
+
+With an open source solution, users continuously contribute to the development of the product and also extend and improve on it as much as they can.
+There are transparent roadmaps, community feedback, and public development processes.
+The eventual fate of certain features are not decided by a single vendor.
```
Comment (AlekSi) [inline website/blog/2024-09-13-why-open-source-important-proprietary-uncertainties.md:53]:
```suggestion
The eventual fate of certain features is not decided by a single vendor.
```

File: website/blog/2024-09-13-why-open-source-important-proprietary-uncertainties.md
Comment (AlekSi) [inline website/blog/2024-09-13-why-open-source-important-proprietary-uncertainties.md:1]: Please bump date to today

**Review by ptrfarkas — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Pull request merged**